app/views/content/subjects/physics/_article.html.erb

```diff
@@ -0,0 +1,198 @@
+        <li class="tag">Energy, forces and wave motion</li>
+        <li class="tag">Electricity, magnetism and electromagnetism</li>
+        <li class="tag">The structure of matter</li>
+        <li class="tag">Atomic structure and space physics</li>
+      </ul>
+    </div>
+ </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <p>
+        16 to 18 year olds (key stage 5):
+      </p>
+    </div>
+
+    <div>
+       <ul class="vertical-tags">
+        <li class="tag">Fields, vectors and scalars</li>
+        <li class="tag">Mechanics and mechanical properties of matter</li>
+        <li class="tag">Electric circuits and waves</li>
+        <li class="tag">Matter, quantum and nuclear physics</li>
+      </ul>
+      </ul>
+    </div>
+ </section>
+
+  <section class="col col-720">
+    <p>
+      <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">Read the national curriculum for more information about physics lessons</a>.
+    </p>
+
+    <p>You can also search the internet for past exam papers to see the type of questions that have been covered.</p>
+
+    <p>
+      You may also need to teach other subjects, particularly for 11- to 16-year-olds and in some cases for post-16. This will vary depending on the individual schools.
+    </p>
+  </section>
+   <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "It's inspiring getting 'wow, that's how it works' from pupils as they grasp a subject that they thought was hard.",
+    name: "Jane Doran, physics and robotics teacher from Coventry, West Midlands",
+    background: "grey",
+    large: true
+  ) %>
+</section>
+
+ <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithOneImageComponent.new(
+      heading: "Next steps into teacher training",
+      colour: "yellow",
+      image_path: "static/content/hero-images/0006.jpg",
+      image_alt: "A class of pupils sitting at desks looking at the camera.",
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Help to become a physics teacher</h3>
+
+    <p>
+      Our experienced teacher training advisers can guide you through everything you need to know to apply for physics teacher training, free of charge. They can help you to:
+    </p>
+    <ul>
+      <li>understand more about physics courses</li>
+      <li>answer questions about fees and funding</li>
+      <li>make an application for a physics course</li>
+      <li>prepare for your teacher training interview</li>
+    </ul>
+    <p><a href="/teacher-training-advisers">Find out about advisers</a></p>
+
+    <p>
+      You can also <a href="https://www.ogdentrust.com/teacher-support/training-to-be-a-teacher/">find out more about becoming a physics teacher from the Ogden Trust</a>, a charitable trust which promotes the teaching and learning of physics.
```

Comment:
This is good to include but does this fit in this section (it might well do!) and right here? I have a slight concern that we might be overshadowing the advisers CTA link with this link directly below it